.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
```

Comment:
Well, this means that you cannot use the trait in a bundle that needs to maintain compatibility with PHP 5.3.